ext/bg/js/search.js

```diff
@@ -69,7 +66,7 @@ class DisplaySearch extends Display {
                         this.wanakanaEnable.checked = false;
                     }
                     this.wanakanaEnable.addEventListener('change', (e) => {
-                        const query = DisplaySearch.getSearchQueryFromLocation(window.location.href) || '';
```

Comment:
Out of context for this change, but the listener callbacks for `this.wanakanaEnable.addEventListener('change', ...)` here and `this.clipboardMonitorEnable.addEventListener('change', ...)` on line 97 should be moved into their own member function definitions on `DisplaySearch`; I think it would make it easier to read the `prepare` function.

Reply:
Yeah, this method exploded out of hand when I added query parser. What's the reason for checking nulls on those elements btw? I just replicated it here.

Reply:
I suppose there's not really a good purpose as it currently stands. While technically it's possible for elements to be removed or modified from other scripts or dev tools, it's probably not necessary to handle these cases. (I may have a habit of doing this due to using other environments in which null checks are more necessary.)

Reply:
Ok. We can look at that again when this is refactored. The nested checks add some extra indentation level.